client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -138,7 +138,7 @@ const renderPriceField = ( { price, currency, ...props } ) => {
 		<FieldsetRenderer
 			inputComponent={ FormCurrencyInput }
 			{ ...price }
-			{ ...props }
+			{ ...omit( props, [ 'names' ] ) }
```

Comment:
Just thinking out loud: another way to do this is destructuring the `names` property into an unused variable in the function params:
```js
// eslint-disable-next-line no-unused-vars
function( { price, currency, names, ...props } ) {}
```
That destructuring already transpiles into something identical to
```js
props = omit( args, [ 'price', 'currency', 'names' ] );
```
So it avoids calling `omit` twice and creating two new objects. The downside is that you have to instruct eslint to ignore the unused variable.

Reply:
It's a great suggestion, but I'll skip - I don't fancy introducing eslint ignores unless really necessary.

Reply:
No worries! I also don't like introducing eslint ignores. I found that ESLint 3.15.0 [introduces](https://github.com/eslint/eslint/pull/7968) a new option for `no-unused-vars`: `ignoreRestSiblings`. That's intended exactly for this use case. Something to consider once we update the version used in Calypso.